jsapp/js/enketoHandler.es6

```diff
@@ -45,31 +44,36 @@ const enketoHandler = {
    *
    * @returns {Promise} Promise that resolves when url is being opened.
    */
-  editSubmission(aid, sid) {
+  openSubmission(aid, sid, action) {
     // we create the tab immediately to avoid browser popup blocker killing it
     this.winTab = window.open('', '_blank');
-    const editPromise = new Promise((resolve, reject) => {
-      if (this._hasEnketoUrl(aid, sid)) {
-        this._openEnketoUrl(aid, sid);
+    let dataInt = dataInterface.getEnketoEditLink;
```

Comment:
Please use a bit more descriptive variable name - I thought that `dataInt` is a variable that is a shortcut to `dataInterface`. Maybe `dataIntMethod`?